src/lib/util/format.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Format a number to a human-readable size string.
+ *
+ * @param      {number}  size    The size.
+ * @return     {string}  Human-readable, formatted version.
+ */
+export function formatSize(size: number): string {
```

Comment:
```suggestion
 * Format a number to a human-readable file size string.
 *
 * @param      {number}  size    The file's size in bytes.
 * @return     {string}  Human-readable, formatted version.
 */
export function formatFileSize(size: number): string {
```
Maybe a bit more specific?